tests/integration/crunchers/test_cruncher_quantile_rolling_windows.py

```diff
@@ -113,12 +114,22 @@ def test_derive_relationship_with_multicolumns(self):
                 "Emissions|CO2", ["Emissions|CH4", "Emissions|HFC|C5F12"]
             )
 
-    def test_relationship_usage(self, simple_df):
+    @pytest.mark.parametrize("use_ratio", [True, False])
+    def test_relationship_usage(self, simple_df, use_ratio, caplog):
```

Comment:
a description here would be useful as this test is relatively long and captures different points.

Reply:
Some added